crypto/evp/evp_enc.c

```diff
@@ -336,7 +302,20 @@ int EVP_EncryptUpdate(EVP_CIPHER_CTX *ctx, unsigned char *out, int *outl,
             return 0;
         }
     }
+
+    /* Past this point inl has to be in bytes. */
     i = ctx->buf_len;
```

Comment:
You can move it even higher... But let's say it's a side note...

Reply:
Sadly I must say, that we have zero test coverage for EVP_CIPH_FLAG_LENGTH_BITS.